miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -1715,7 +1715,7 @@ public static LibraryAliquotDto asDto(@Nonnull LibraryAliquot from, boolean incl
         if (sample.getProject() != null) {
           setLong(dto::setProjectId, project.getId(), true);
           setString(dto::setProjectName, project.getName());
-          setString(dto::setProjectShortName, project.getShortName());
+          setString(dto::setProjectShortName, project.getCode());
```

Comment:
Please modify `LibraryAliquotDto` to change `get/setProjectShortName` -> `get/setProjectCode`